manual/source/topic/format.rst

```diff
@@ -510,22 +510,26 @@ Object format introspection
     class's discretion: the stepper function must handle this
     case.
 
-    .. note::
-
-        This function is intended for heap analysis, tuning, and
-        debugging, not for frequent use in production.
-
     .. warning::
 
         If a garbage collection is currently in progress (that is, if
         the arena is in the :term:`clamped <clamped state>` or
         :term:`unclamped state`), then only objects that are known to
-        be currently valid are visited.
+        be currently valid are visited, and the callback function must
```

Comment:
Start a new sentence before "the callback function" otherwise it appears to be guarded by the "if a garbage collection" condition.